partner_internal_code/demo/res.partner.csv

```diff
@@ -16,31 +16,27 @@ base.res_partner_3,14
 base.res_partner_address_5,15
 base.res_partner_4,16
 base.res_partner_address_7,17
```

Comment:
Dont need to remove ``portal.demo_user0_res_partner`` instead replace it  with ``base.demo_user0_res_partner`` which is the same record only that the one in version 11.0 has been moved from module